base/CMakeLists.txt

```diff
@@ -68,6 +68,7 @@ set(BASE_SOURCES
     all-all/dig090.png
     all-all/dig091.png
     all-all/dig093.png
+    all-all/dmxopl.op2
```

Comment:
License?

Reply:
Added to readme: https://github.com/fabiangreffrath/woof/pull/1988/commits/32e918a7b47d8c75f1c57155e54df49eb9022087